src/garage/tf/misc/tensor_utils.py

```diff
@@ -63,6 +64,14 @@ def filter_valids(t, valid, name='filter_valids'):
         t, tf.cast(tf.round(valid), tf.int32), 2, name=name)[1]
 
 
+def normalize_pixel_observation(env_spec, obs):
```

Comment:
I believe we already have this, check out `garage.misc.tensor_utils.normalize_pixel_batch`

Reply:
Yep though that takes in a batch of observations and returns a list of normalized observations since it's used by the sampler. This function just accepts one observation. Although `garage.misc.tensor_utils.normalize_pixel_batch` can be rewritten to call `normalize_pixel_observation` in a loop.

Reply:
I see. Indeed, the difference is probably that one deals with python array while one deals with tf.Tensor.

Reply:
Would be great if you can provide more information in the docstring. Please refer to [this link](https://github.com/rlworkgroup/garage/blob/master/CONTRIBUTING.md) for guide.

Reply:
Thanks, updated. I also followed the docstring format in `garage.misc.tensor_utils.normalize_pixel_batch`.